frontend/app/lib/api.ts

```diff
@@ -1,28 +1,37 @@
-import { apiUrl } from "../utils/env";
+import { apiUrl } from "../utils/env"
```

Comment:
Not sure why this `;` was removed, but we might want to create some unified linting settings to avoid these.